### PR TITLE
Add degreeOfParallelismPercent and hierarchyQueueNode to job contract

### PR DESCRIPTION
**Pull request by hosunmsft**
## Description

1. Add a property "degreeOfParallelismPercent" as a supplementary to the existing property "degreeOfParallelism" to the CreateJobParameters class and the UpdateJobParameters class
2. Add two fields, "DegreeOfParallelismPercent" and "HierarchyQueueNode" to JobInformationBasic class

Corresponding Spec RPs 
[https://github.com/Azure/azure-rest-api-specs/pull/3871](url)
[https://github.com/Azure/azure-rest-api-specs/pull/3904](url)
[https://github.com/Azure/azure-rest-api-specs/pull/3942](url)

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by hosunmsft** [comment]
The corresponding spec change is in the following PRs [https://github.com/Azure/azure-rest-api-specs/pull/3871](url) [https://github.com/Azure/azure-rest-api-specs/pull/3904](url) [https://github.com/Azure/azure-rest-api-specs/pull/3942](url)

File: src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataLake.Analytics</PackageId>
     <Description>Provides Data Lake Analytics account, job and catalog management capabilities for Microsoft Azure.</Description>
-    <Version>3.5.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj]: I believe 2 SDKs were modified here, please bump versions in boths csproj files

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Please fix CI build failures

File: src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj (hunk 1 above)
Comment (hosunmsft) [reply src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj]: No, only DataLakeAnalyticsJobManagementClient is modified. Some changes are automatically generated without any change to the API Spec.

**Comment by hosunmsft** [comment]
Hi Deepak,

Thanks for your comments.
I just updated the PR to fix the CI build failure. Please take another look.
https://github.com/Azure/azure-sdk-for-net/pull/4788

For your other comment, there’s only one SDK changed in this PR -Microsoft.Azure.Management.DataLake.Analytics.

Thanks
Hongfeng

File: src/SDKs/_metadata/datalake-analytics_data-plane.txt [hunk 2]
```diff
@@ -1,38 +1,14 @@
...
-
-> autorest@2.0.4280 preinstall C:\Program Files\nodejs\node_modules\.staging\autorest-2f6de348
-> node ./preinstall-check
-
-C:\Program Files\nodejs\autorest -> C:\Program Files\nodejs\node_modules\autorest\dist\app.js
-C:\Program Files\nodejs
-`-- autorest@2.0.4280 
-
-
 AutoRest installed successfully.
```
Comment (dsgouda) [inline src/SDKs/_metadata/datalake-analytics_data-plane.txt]: @hosunmsft does this file get modified too when the code is regenerated

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**